index.html

```diff
@@ -5021,55 +5021,28 @@ <h3 id="playback-AudioBufferSourceNode">
             The description of the algorithm is as follows:
           </p>
           <pre>
-// The AudioBuffer and context employed by this node
-var buffer;
-var context;
+let buffer;  /* AudioBuffer employed by this node */;
+let context; /* AudioContext employed by this node */;
 
 // The following variables capture attribute and AudioParam values for the node.
-// They are updated k-rate, except for buffer.
-var loop;
-var detune;
-var loopEnd;
-var loopStart;
-var playbackRate;
-
-// State variables for the node's playback
-var start = 0;
-var offset;
-var stop = Infinity;
-
-var bufferTime = 0;
-var started = false;
-var enteredLoop = false;
-var dt;
-
-// Handle invocation of start method call
-function handleStart(when, pos, duration) {
-  if (arguments.length &gt;= 1) {
-    start = when;
-  }
-  offset = pos;
-  if (arguments.length &gt;= 3) {
-    stop = when + duration;
-  }
-}
+// They are updated on a k-rate basis, prior to each invocation of process().
+let loop, detune, loopStart, loopEnd, playbackRate;
```

Comment:
I know people disagree on this, but I hate the style that declares a bunch of vars all on one line.

I defer to others on this.